service/server.go

```diff
@@ -127,6 +128,15 @@ func (s *server) ListenAndServe(ctx context.Context) chan []error {
 	wg := new(sync.WaitGroup)
 	wg.Add(2)
 
+	currentUser, err := user.Current()
+	if err != nil {
+		glg.Error(errors.Wrap(err, "failed to get current user"))
```

Comment:
@hiyosi 

> Should the error be handled like other codes?
sure. actually should be `warning` instead. thanks.

> Is it expected to continue processing even if an error is occurred.
yes, because it won't happen most of the times & don't want to bother core logics just because a new logger function is added.

Reply:
Handled in: https://github.com/AthenZ/garm/pull/41/commits/92b2b310e8d9cbccd3edbcc28abb4a65a533faed
Intentionally removed  `"failed to get current user"` to follow other warning styles